pkg/tracing/traceparent.go

```diff
@@ -73,7 +73,13 @@ func AddSpanFromTraceparentAttribute(ctx context.Context, name string, event clo
 	if !ok {
 		return ctx, fmt.Errorf("extension attribute %q's value was not a string: %T", traceparentAttribute, tps)
 	}
-	sc, err := parseTraceparent(tps)
+	return ParseSpanFromTraceparentAttribute(ctx, name, tps)
+}
+
+// ParseSpanFromTraceparentAttribute works the same as AddSpanFromTraceparentAttribute, but takes
+// the extension value as function parameter
+func ParseSpanFromTraceparentAttribute(ctx context.Context, name string, tp string) (context.Context, error) {
```

Comment:
I need this method to extract the traceparent directly from the kafka message in KafkaChannel